config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
   devise_for :users, controllers: { omniauth_callbacks: 'omniauth_callbacks', registrations: 'registrations' }
   root to: 'users#index'
+  resources :users, only: %i[edit show index]
+  put 'update', action: :update, to: 'users#update'
```

Comment:
Why do you use `put update`?

Reply:
> Why do you use `put update`?

Because I need to get update_path and send it to props for ajax request